examples/features/metadata_interceptor/server/main.go

```diff
@@ -0,0 +1,139 @@
+/*
+ *
+ * Copyright 2022 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Binary server is an example server.
+package main
+
+import (
+	"context"
```

Comment:
Seems like we need some reordering of the imports here to pass `vet` tests. Try 
```shell
goimports examples/features/metadata_interceptor/server/main.go
```
to see how the file should look like

Reply:
@arvindbr8 , thank you very much for your response, I have updated the reordering of the imports through `goimports`.